Gradle Build failed on commit 675e397

Fix gradle build errors in VisionIOPhotonVision.java

`/home/runner/work/GS25Rewrite/GS25Rewrite/src/main/java/frc/robot/vision/VisionIOPhotonVision.java:79: error: cannot find symbol
  			inputs.tagObservations = new TagObservation[tagObservations.size()];
  			      ^
    symbol:   variable tagObservations
    location: variable inputs of type VisionIOInputs
  /home/runner/work/GS25Rewrite/GS25Rewrite/src/main/java/frc/robot/vision/VisionIOPhotonVision.java:81: error: cannot find symbol
  				inputs.tagObservations[i] = tagObservations.get(i);
  				      ^
    symbol:   variable tagObservations
    location: variable inputs of type VisionIOInputs`

--- a/src/main/java/frc/robot/Features.java
+++ b/src/main/java/frc/robot/Features.java
@@ -2,7 +2,6 @@ package frc.robot;
 
 public class Features {
 	public static final boolean PHOTON_POSE_ESTIMATOR_ENABLED = true;
-	public static final boolean USE_TEST_FIELD = true;
-
+	public static final boolean USE_TEST_FIELD = false;
 	public static final boolean MAPLE_SIM_ENABLED = true;
 }

--- a/src/main/java/frc/robot/vision/VisionIO.java
+++ b/src/main/java/frc/robot/vision/VisionIO.java
@@ -119,11 +119,6 @@ public interface VisionIO {
 			PoseObservationType type) {
 	}
 
-	public static record TagObservation(
-		double timestamp,
-		double distanceToTag,
-		int tagID) {
-	}
 
 	enum PoseObservationType {
 		MEGATAG_1,

--- a/src/main/java/frc/robot/vision/VisionIOPhotonVision.java
+++ b/src/main/java/frc/robot/vision/VisionIOPhotonVision.java
@@ -49,7 +49,6 @@ public class VisionIOPhotonVision implements VisionIO {
 		// Read new camera observations
 		Set<Short> tagIds = new HashSet<>();
 		List<PoseObservation> poseObservations = new LinkedList<>();
-		List<TagObservation> tagObservations = new LinkedList<>();
 
 		for (var result : camera.getAllUnreadResults()) {
 			// Update latest target observation
@@ -64,21 +63,6 @@ public class VisionIOPhotonVision implements VisionIO {
 				inputs.setLatestTargetObservation(
 					new TargetObservation(new Rotation2d(), new Rotation2d())
 				);
-			}
-
-			// Add tag observations
-			if (result.hasTargets()) {
-				for (var target : result.getTargets()) {
-					double distance = target.getBestCameraToTarget().getTranslation().getNorm();
-					double timestamp = result.getTimestampSeconds();
-					int tagID = target.getFiducialId();
-
-					tagObservations.add(new TagObservation(timestamp, distance, tagID));
-				}
-			}
-			inputs.tagObservations = new TagObservation[tagObservations.size()];
-			for (int i = 0; i < tagObservations.size(); i++) {
-				inputs.tagObservations[i] = tagObservations.get(i);
 			}
 
 			// Add pose observation

--- a/src/main/java/frc/robot/vision/VisionIOPhotonVisionSim.java
+++ b/src/main/java/frc/robot/vision/VisionIOPhotonVisionSim.java
@@ -16,7 +16,6 @@ package frc.robot.vision;
 import edu.wpi.first.math.geometry.Pose2d;
 import edu.wpi.first.math.geometry.Transform3d;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-import frc.robot.Constants.SimConstants;
 
 import static frc.robot.Constants.VisionConstants.TAG_LAYOUT;
 

